packages/http-client-python/emitter/src/emitter.ts

```diff
@@ -163,6 +163,7 @@ async function onEmitMain(context: EmitContext<PythonEmitterOptions>) {
   }
 
   for (const [key, value] of Object.entries(resolvedOptions)) {
+    if (key === "license") continue; // skip license since it is passed in codeModel
```

Comment:
Since license is an object contains `name`/`header`/..., run_tsp.py doesn't support pass object for now. We could skip it directly because we have passed the handled info by TCGC through codemodel.